### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by calin-coj**
Unit tests for the catalog endpoints
## Description

Added some preliminary unit tests for the catalog endpoints. In future PRs, will need to revisit the response codes (not everything should return 200)  and check the payload integrity as well 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I am happy with the test coverage for the newly added code.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.

- [ ] `fixes #issue_number` or `closes #issue_number` in the body of the pull request.

### CODE REVIEW

**Comment by calin-coj** [comment]
> Think it's worth testing with something like https://taverntesting.github.io/ ?

I'm looking at it right now, sounds interesting.

File: tests/test_endpoints.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import pytest
+from zia_connect.app import init_app
+
+
+@pytest.fixture
+def client(scope="session"):
+    app = init_app()
+    app.testing = True
+    flask_client = app.test_client()
```
Comment (markuzo) [inline tests/test_endpoints.py]: Awesome. Maybe we should give Flask more credit than we do.

**Comment by calin-coj** [comment]
> Think it's worth testing with something like https://taverntesting.github.io/ ?

I've looked at tavern, and I feel like there are 100 different tools that do the exact same thing (see postman). I don't feel the need for it right now, but it's not like I'm a rugged API tester. If you have any insight on why it's clearly better than the regular pytest, the sure, let's go for it

**Comment by ppop** [comment]
I'm no rugged API tester either, just thought it's worth mentioning since you're doing the tests now.

**Pull request merged**